### PR TITLE
Create dev environment for VSCode

### PR DESCRIPTION
**Pull request by vvmruder**

### CODE REVIEW

**Comment by jwkaltz** [comment]
@vvmruder I created a maintenance release 1.9.2 now, and a branch v1
So feel free now to overwrite the Makefile, the ci.yaml, docker-compose etc. in this pull request and in master branch.

**Comment by jwkaltz** [comment]
@vvmruder and please increase the version number to 2.0.0.

**Comment by vvmruder** [comment]
@jwkaltz Thanks for ping. I added the new version. I also removed all "old" stuff to start with clean make and testing.

**Comment by vvmruder** [comment]
@jwkaltz I think we are done here. @voisardf approved it is working in his environment on the ports configured.

Only Question is now about the Codacy check. I Don't know what it is about and how to fix it.

File: .devcontainer/devcontainer.json [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+    "ms-python.vscode-pylance",
+    "mtxr.sqltools",
+    "mtxr.sqltools-driver-pg"
+  ],
+
+  // Use 'forwardPorts' to make a list of ports inside the container available locally.
+  "forwardPorts": [6543, 5432],
+
+  // Use 'postCreateCommand' to run commands after the container is created.
+  "postCreateCommand": "make clean-all install",
```
Comment (vuilleumierc) [inline .devcontainer/devcontainer.json:50]:
```suggestion
  "postCreateCommand": "make clean-all install"
```
Have you tried this for the Codacy Check? (I cannot test here, the test does not start?)

Comment (vvmruder) [reply .devcontainer/devcontainer.json]: This changed nothing.

Comment (vuilleumierc) [reply .devcontainer/devcontainer.json]: It did fix the `Unexpected character ('}' (code 125)): was expecting double-quote to start field name` failure (I thought that was was you were trying to fix)

Comment (vvmruder) [reply .devcontainer/devcontainer.json]: OK. Thanks. I decided with Wolfgang to exclude the `.devcontainers` folder from the codacy checks

**Review by kdeininger — APPROVED** [review]

**Pull request merged**